src/components/Network.tsx

```diff
@@ -97,16 +97,21 @@ export const NetworkStats = () => {
   return (
     <>
       <Box className={classes.statsEntry}>
-        Bridged blocks:
+        Finilised blocks:
         <span>
-          {dbSource.bestBridgedFinalizedBlock} / {dbSource.bestBlockFinalized}
+          {dbSource.bestBlockFinalized} / {dbSource.bestBlock}
```

Comment:
In all the dbsource cases below you must ensure that dbSource exists (add a ? before each dot). If it does not - then that could break the page

Reply:
good catch

Reply:
if you use vscode you can set to format the file during the onSave and adding the ? is one of the things that are added automatically by you.

Reply:
I do not like to let things do automatically stuff for me - I like to have control of the code :)